docs/pulse/introduction.md

```diff
@@ -4,32 +4,69 @@ sidebar_label: Introduction
 
 ![image](https://user-images.githubusercontent.com/1315028/134992035-1bfa67f2-73a0-4b88-ac1d-688fa6ef0b33.png)
 
-## Export Report
+## Metrics Drill-Down
+
+A tooltip with key statistics and deeper information is shown if you hover over a metric in Pulse.
+
+![image](https://user-images.githubusercontent.com/1315028/131383593-384225bc-abbd-483f-a45a-3280d8bf5941.png)
+
+ - **Group**: The name of the group of users.  For Feature Gates, the "Pass" group is considered the test group while the "Fail" group is the control.  In experiments, these will be the variant names.
+ - **Units**: The number of observations included in the metric.  Most metrics report a value for each exposed user on a daily basis.  A user who has been in the experiment for 7 days will produce 7 observations (units).
+ - **Mean**: Average of the metric across the units (observations).
+ - **Std**: Standard error of the mean estimate for this metric.
+ - **Abs Delta**: The absolute difference of the Mean between test groups i.e. Test Mean - Fail Mean.
+ - **Delta %**: Relative difference of the Mean i.e. 100% x (Pass mean – Fail mean) / Fail mean
+ - **P-Value**: The (two-sided) probability of achieving the observed difference (or one more extreme) assuming there is no difference and the experiment has no effect (groups are equivalent). In classical hypothesis testing, a low p-value means that the chance of obtaining such a result is rare and if it's below a preset threshold, we can conclude that there must be a difference or real effect.
+
+# Best Practices and Avoiding False Positives
+
+We have some suggestions on how to interpret Pulse in a scientifically-sound way:
+1. Metrics shouldn't be cherry picked and evaluated as "good" or "bad" without context.  Context matters a lot, and any statistically-significant result needs to have a plausible explanation.
+2. Any statistically-significant result should ideally be backed by either a really low p-value, or by corroborating (ideally independent) metrics.
+3. You should have a hypothesis in mind before viewing Pulse.  What are the metric(s) you expect to shift due to the change you made?  What else could have happened?  What are signs it has gone wrong?
+4. Seeing multiple effects, with low p-values that are consistent with a plausible story lends credibility that the observed effects are real.
+5. Expect to see false positives and be suspicious of statistically significant results.  Especially those with borderline p-values.
```

Comment:
perhaps note that the more metrics you have, the more likely you are to have 1 or more report as a stat sig change when they are not